docs/variants/novacustom_ns5x_adl/building-manual.md

```diff
@@ -0,0 +1,58 @@
+# Building manual
```

Comment:
Isn't this redundant?

Reply:
I hoped it is. I wanted to take over this PR and unify this, removing 4 building manuals and replacing them with a single one, as a first step.

Reply:
Doc removed.